lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -72,7 +72,8 @@ class OffscreenImages extends Audit {
     const url = URL.getDisplayName(image.src, {preserveQuery: true});
     const totalPixels = image.clientWidth * image.clientHeight;
     const visiblePixels = this.computeVisiblePixels(image.clientRect, viewportDimensions);
-    const wastedRatio = 1 - visiblePixels / totalPixels;
+    // Treat images with 0 area as if they're offscreen
```

Comment:
Agreed. Don't want to derail this fix. 

@patrickhulce maybe file a bug capturing this info? We can improve the audit in the future.